xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-validation/xwiki-platform-model-validation-api/src/main/java/org/xwiki/model/validation/edit/EditConfirmationCheckerResult.java

```diff
@@ -39,16 +41,35 @@ public class EditConfirmationCheckerResult
 
     private final boolean isError;
 
+    private final Serializable cache;
+
     /**
      * Constructs a new object with the specified message and error status.
+     * This constructor initializes the {@code cache} with a {@code null} value,
+     * meaning that the result can't be forced.
```

Comment:
It cannot be forced at all, or if it's forced then the user will be asked again?

Reply:
Updated, see https://github.com/xwiki/xwiki-platform/pull/2595/files#diff-bf2ed29667be98ddff3766113c301ce068688c7f6d10af4ef4f9e2c1f5996776R51